var/classes/objectbricks/PaymentProviderHeidelPay.php

```diff
@@ -31,7 +31,6 @@
    'layoutDefinitions' => 
   Pimcore\Model\DataObject\ClassDefinition\Layout\Panel::__set_state(array(
      'fieldtype' => 'panel',
-     'labelWidth' => 100,
```

Comment:
remove heidelpay completely since it was renamed to unzer?

Reply:
removed